backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1650,3 +1650,15 @@ databaseChangeLog:
                   defaultValueNumeric: 200
                   constraints:
                     nullable: false
+  - changeSet:
```

Comment:
This is a really good question, and I wasn't well-prepared to answer it at the time, because there is no prior art for one-to-many relations on _patient_, which eventually gets serialized as JSON inside a TestEvent. A similar problem exists for `multiple telephone numbers`.

I'm going to go adjust the data model to support multiple tribal affiliations, but leave the front end alone (to be addressed in #770)

Reply:
So, I went back and forth on this a number of times – whether or not to add a join table, how to minimize impacts on the front end, etc.

What I settled on was to add a `jsonb` field, `tribal_affiliation`, to Person. This serializes easily but does not add any additional postgres query complexity for the areas that might have been impacted by performance here.

When implementing #770, the developer will need to update the relevant GQL mutations to accept `[String]` and update PersonService accordingly.